nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -909,6 +910,12 @@ mod test {
             "2 -> 3 (expect new NTP zone on new sled):\n{}",
             diff.display()
         );
+        let diffus_diff = blueprint3.diff(&blueprint2);
+        println!("_______DIFFUS __________\n");
+        let change = diffus_diff.change();
+        println!("{:#?}", diffus_diff);
+        println!("{:#?}", change);
+        println!("________DIFFUS __________\n");
```

Comment:
Just checking - intended to leave this here?

Reply:
I just removed it. Thanks!